src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Value.ReadTests.cs

```diff
@@ -654,6 +654,12 @@ public static void DateOnly_Read_Failure(string json, bool addQuotes = true)
         }
 
         [Theory]
+        [InlineData("1:2", "01:02")]
```

Comment:
It seems the original change in https://github.com/dotnet/runtime/pull/102091/files didn't add test cases for single-digit hour and minute values. Would it be possible to add a few test cases for that as well?

Reply:
Sure! See new commit